packages/verification/src/models/v1/flashcall-verification-status-response/flashcall-verification-status-response.ts

```diff
@@ -0,0 +1,31 @@
+import { CallResult, ReasonEnum, SourceEnum, VerificationStatusEnum } from '../enums';
+import { Identity } from '../identity';
+import { VerificationPriceCall } from '../verification-price-call';
+
+export interface FlashCallVerificationStatusResponse {
+
+  /** The unique ID of the verification request. */
+  id?: string;
+  /** The method of the verification request. This will always be `flashCall`. */
+  method?: 'flashcall';
+  /** The status of the verification request. */
+  status?: VerificationStatusEnum;
+  /** Displays the reason why a verification has `FAILED`, was `DENIED`, or was `ABORTED`. */
+  reason?: ReasonEnum;
+  /** Shows whether the call is complete or not. */
+  callComplete?: boolean;
+  /** The result of the call. */
+  callResult?: CallResult;
```

Comment:
Not present onto online doc

Reply:
I've put the 2 fields in the previous response 😁